calour/heatmap/heatmap.py

```diff
@@ -164,8 +164,11 @@ def heatmap(exp: Experiment, sample_field=None, feature_field=None,
     ----------
     sample_field : str or None, optional
         The field of sample metadata to display on the x-axis or None (default) to not show x axis.
-    feature_field : str or None, optional
-        The field of feature meadata to display on the y-axis or None (default) to not show y axis.
+    feature_field : str or None or False (optional)
```

Comment:
I still don't get why we need to set a default feature field? allowing both `None` and `False` is confusing to users.

Reply:
The idea is that for ms data, we want to have ticks for the y-axis with the metabolite name by default, whereas for amplicon we don't want the y axis ticks by default.
I don't think it will be too confusing if explained in the doc? 
alternatively, we can remove the default ytick for ms, and have users supply it explicitly, but then less convenient for the naive mass-spec user?

Reply:
I prefer the latter, making the API behavior more consistent. I would argue for naive mass spec users, MZ yticklabel probably don't help them anyway? you can add a doc for feature_metadata, saying for ms exp, you can supply MZ.

I don't like this because 1) None and False is confusing even with doc (I often can't recall which is to disable the yticklabel); 2) it makes the classes have yet another ad hoc attribute of feature metadata.

Other than this, I am good to merge.